fluentd/configs.d/openshift/output-applications.conf

```diff
@@ -0,0 +1,5 @@
+<match **>
+   @type copy
+   @include output-es-config.conf
+   @include ../dynamic/es-copy-config.conf
```

Comment:
I suggest to have an `@include output-extra-*.conf` here - then the user could add a config map for `output-extra-splunk.conf` or `output-extra-fluentd-secure-forward.conf` or whatever